lib/nimble_options.ex

```diff
@@ -127,8 +127,9 @@ defmodule NimbleOptions do
       ...>   ]
       ...> ]
       ...>
-      ...> NimbleOptions.validate(config, schema)
-      {:error, "(in options [:producer]) required option :module not found, received options: [:concurrency]"}
+      ...> {:error, %NimbleOptions.ValidationError{} = error} = NimbleOptions.validate(config, schema)
+      ...> Exception.message(error)
+      "(in options [:producer]) required option :module not found, received options: [:concurrency]"
```

Comment:
Can we move the path to the end? Thanks!

Reply:
Done!